Syslogv2 was removed from xsiam marketplace (#27278)
demisto/content: Packs/Syslog/ReleaseNotes/2_0_16.md

#### Integrations

##### Syslog v2
<~XSIAM>

> - Updated the integration to upload to XSOAR Marketplace, not to XSIAM. XSIAM users should use BrokerVM module instead.  

Reviewer: ```suggestion
- Removed from the XSIAM marketplace. In XSIAM, use the `BrokerVM` module instead. 
```
